python/dlisio/plumbing/basicobject.py

```diff
@@ -17,6 +17,19 @@ class BasicObject():
 
     attributes = {}
     linkage    = {}
+    """dict: Shows how to to interpret link objects stored in refs.
```

Comment:
Could maybe be a bit clearer. "Linkage defines the rules for how each attribute links to other objects"?